src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -18,15 +18,16 @@
 
     <button type="button"
             class="navbar-toggler"
-            (click)="isCollapsed = !isCollapsed">
+            (click)="toggleRightSidebar()">
       <span i18n
             class="sr-only">Toggle navigation</span>
       <span class="">
         <i class="fa fa-navicon fa-lg"></i>
       </span>
     </button>
 
-    <div class="collapse navbar-collapse">
+    <div class="collapse navbar-collapse show"
```

Comment:
Thank for your contribution!
I observe that now when expanding the content it hides the landing page:
![Screenshot from 2021-05-31 16-17-45](https://user-images.githubusercontent.com/6780162/120212256-0bf9b680-c232-11eb-9015-7ac300b2128d.png)

is it possible to achieve the same behavior as the menu bar?